family2family/src/main/java/org/cbtf/f2f/model/Users.java

```diff
@@ -15,6 +17,17 @@
 
     private Boolean active;
 
+    @OneToOne
+    @JoinColumn(name = Contact.COLUMN_PRIMARY_EMAIL)
+    private Contact contact;
+
+    public Users(String id, Boolean admin, Boolean active, Contact contact) {
```

Comment:
http://localhost:8080/internal-api/Users errors. Complains about not having a default constructor.